src/main/java/com/epam/tc/hw9api/utils/PropertiesReader.java

```diff
@@ -0,0 +1,29 @@
+package com.epam.tc.hw9api.utils;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Properties;
+
+public class PropertiesReader {
+
+    public static Properties properties;
+
+    public PropertiesReader() {
+    }
+
+    public static String getProperty(String getData) {
```

Comment:
read from properties once in the static initialization block, CLOSE streams (or use try with resources)

Reply:
try with resources is added